cli/app.go

```diff
@@ -416,12 +422,12 @@ var app = &cli.App{
 				{
 					Name:      "database",
 					Usage:     "interact with a MongoDB Atlas Data Federation instance",
-					UsageText: "viam data database [other options]",
+					UsageText: createUsageText("data database", nil, true),
```

Comment:
Didn't add usage texts for commands with no required flags very often, but this `UsageText` was already here and I thought I'd use the function.

Reply:
ah okay, this is the sort of command I was thinking of when I made [this comment](https://github.com/viamrobotics/rdk/pull/3570/files#r1486367157).